### PR TITLE
edited whitespaces to retrigger travis build

### PR DESCRIPTION
**Pull request by matthdsm**
* [x] I have read the guidelines above.
* [ ] This PR adds a new recipe.
* [] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

edited some whitespace to retrigger build

### CODE REVIEW

File: recipes/umi_tools/meta.yml [hunk 1]
```diff
@@ -1,16 +1,13 @@
...
   version: "0.2.3"
-
 source:
   fn: umi_tools-0.2.3.tar.gz
   url: https://pypi.python.org/packages/de/53/6fa261cfe6f5b3272795849a97ac53898f13849522d0f8fa7e4dd0696e15/umi_tools-0.2.3.tar.gz
   md5: 9726997f170167013bbff3071b25afb6
-
 build:
   number: 0
   skip: False
```
Comment (bgruening) [inline recipes/umi_tools/meta.yml]: This one skips the build entirely. Can you remove it?

**Comment by bgruening** [comment]
Ok, it is still not in the channel. I don't know why. Need to dig deeper, sorry.

**Comment by bgruening** [comment]
@johanneskoester @daler any idea?

**Comment by johanneskoester** [comment]
The file is called meta.yml instead of meta.yaml.

**Comment by johanneskoester** [comment]
You see the error immediately if you try to build locally (conda prints: `No meta.yaml files found in /home/johannes/scms/bioconda-recipes/recipes/umi_tools`). However, our build scripts just ignore directories which don't contain meta.yaml.

**Comment by bgruening** [comment]
Uh, nice catch!

**Comment by matthdsm** [comment]
rookie mistake! Very sorry, fixed in #2126 

M

**Comment by johanneskoester** [comment]
No problem! Now we know it and spot it immediately next time. Would be nice to have a linting system for this kinds of stuff. Maybe we can use the one from conda-forge.

**Comment by bgruening** [comment]
@johanneskoester I updated: https://github.com/bioconda/bioconda-recipes/issues/1860

**Pull request merged**